ui/src/components/inputs/MonacoEditor.vue

```diff
@@ -463,9 +473,31 @@
 
     const disposeCompletions = ref<() => void>();
 
+
+    const errorMarkers = ref<any[]>([]);
+
+    function updateErrorMarkers() {
+        const model = localEditor?.getModel();
+        if (model) {
+            errorMarkers.value = monaco.editor.getModelMarkers({resource: model.uri});
+        } else {
+            errorMarkers.value = [];
+        }
+    }
+
+    function goToError(err: any) {
+        if (localEditor) {
+            localEditor.setPosition({lineNumber: err.startLineNumber, column: err.startColumn});
+            localEditor.focus();
+        }
+    }
+
     const pluginsStore = usePluginsStore();
 
-    const prefix = computed(() => props.schemaType ? `${props.schemaType}-` : "");
+
+
+    const prefix = computed(() => props.schemaType ? `${propchemaType}-` : "");
```

Comment:
This line is producing console error an preventing me to continue. Could you please amend it and make sure there are no more errors in the console regarding this change?

![image](https://github.com/user-attachments/assets/ee91a9e8-9354-42f8-bd9a-2bd3d2824069)